<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>passCode</class>
 <widget class="QWidget" name="passCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>passCode</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lbl_title">
       <property name="minimumSize">
        <size>
         <width>201</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Passcode</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>548</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>338</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbl_info">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>338</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="minimumSize">
        <size>
         <width>191</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>188</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_num">
       <property name="minimumSize">
        <size>
         <width>351</width>
         <height>75</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="minimumSize">
        <size>
         <width>211</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>208</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="minimumSize">
      <size>
       <width>43</width>
       <height>23</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>112</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2" rowspan="3">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>491</width>
       <height>311</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="three">
      <property name="geometry">
       <rect>
        <x>324</x>
        <y>9</y>
        <width>158</width>
        <height>73</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>158</width>
        <height>73</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="four">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>83</y>
        <width>158</width>
        <height>73</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>158</width>
        <height>73</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="five">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>83</y>
        <width>157</width>
        <height>73</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>157</width>
        <height>73</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="six">
      <property name="geometry">
       <rect>
        <x>324</x>
        <y>83</y>
        <width>158</width>
        <height>73</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>158</width>
        <height>73</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="seven">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>157</y>
        <width>158</width>
        <height>72</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>158</width>
        <height>72</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="eight">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>157</y>
        <width>157</width>
        <height>72</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>157</width>
        <height>72</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nine">
      <property name="geometry">
       <rect>
        <x>324</x>
        <y>157</y>
        <width>158</width>
        <height>72</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>158</width>
        <height>72</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_black">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>230</y>
        <width>158</width>
        <height>73</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>158</width>
        <height>73</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="zero">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>230</y>
        <width>157</width>
        <height>73</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>157</width>
        <height>73</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_del">
      <property name="geometry">
       <rect>
        <x>324</x>
        <y>230</y>
        <width>158</width>
        <height>73</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>158</width>
        <height>73</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
     <widget class="QPushButton" name="one">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>9</y>
        <width>158</width>
        <height>73</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>158</width>
        <height>73</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="two">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>9</y>
        <width>157</width>
        <height>73</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>157</width>
        <height>73</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Helvetica Neue</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="btn_forgot">
     <property name="minimumSize">
      <size>
       <width>121</width>
       <height>81</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Forgot Passcode?</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="btn_done">
     <property name="minimumSize">
      <size>
       <width>121</width>
       <height>81</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="btn_back">
     <property name="minimumSize">
      <size>
       <width>121</width>
       <height>81</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>frame</zorder>
  <zorder></zorder>
  <zorder>btn_back</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>horizontalSpacer_4</zorder>
  <zorder>horizontalSpacer_5</zorder>
  <zorder>horizontalSpacer_6</zorder>
  <zorder>btn_done</zorder>
  <zorder>btn_forgot</zorder>
  <zorder>horizontalSpacer_3</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
